src/abb_system_position_only.cpp

```diff
@@ -233,7 +223,7 @@ return_type ABBSystemPositionOnlyHardware::read()
 {
   RCLCPP_INFO(rclcpp::get_logger("ABBSystemPositionOnlyHardware"), "read()");
 
-  abb::egm::wrapper::Input tmp_input;  // TODO(seng): function no longer const, change this
+  abb::egm::wrapper::Input tmp_input;
```

Comment:
Rename to `temp_input`, just to stay consistent

Same for `temp_current_positions` below